src/App.js

```diff
@@ -1,14 +1,24 @@
+import { useLocation } from "react-router-dom";
+import { ToastContainer } from "react-toastify";
 import "./App.css";
 import Header from "./components/Header";
 import Sidebar from "./components/Sidebar";
+import { useTheme } from "./contexts/ThemeContext";
 import Router from "./router/Router";
 
 function App() {
+  const location = useLocation();
+  const { theme } = useTheme();
   return (
     <div className="App">
-      <Header />
-      <Sidebar />
+      {location.pathname !== "/login" && location.pathname !== "/signup" && (
```

Comment:
You can make this logic shorter by storing the pathname in a variable and then comparing with the variable instead of doing location.pathname each time.